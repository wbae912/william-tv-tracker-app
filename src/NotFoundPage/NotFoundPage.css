.not-found {
  text-align: center;
  color: white;
  font-family: 'Montserrat', sans-serif;
  padding-top: 100px;
}

.not-found-h1 {
  font-weight: bold;
  font-size: 20px;
  margin: 0;
}

@media only screen and (min-width: 750px) {
  .not-found-h1 {
    font-size: 24px;
  }
}

@media only screen and (min-width: 1300px) {
  .not-found-h1 {
    font-size: 30px;
  }
}