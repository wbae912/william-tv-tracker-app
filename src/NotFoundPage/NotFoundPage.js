import React, { Component } from 'react';
import './NotFoundPage.css';

export default class NotFoundPage extends Component {
  render() {
    return (
      <div className="not-found" role="main">
        <h1 className="not-found-h1">Page Not Found!</h1>
      </div>
    )
  }
}