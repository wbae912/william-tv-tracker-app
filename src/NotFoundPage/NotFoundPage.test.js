import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundPage from './NotFoundPage';
import { MemoryRouter } from 'react-router';
import renderer from 'react-test-renderer';

describe('NotFoundPage component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MemoryRouter><NotFoundPage /></MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('renders UI as expected', () => {
    const tree = renderer
      .create(<MemoryRouter><NotFoundPage /></MemoryRouter>)
      .toJSON();
      expect(tree).toMatchSnapshot();
  })
})