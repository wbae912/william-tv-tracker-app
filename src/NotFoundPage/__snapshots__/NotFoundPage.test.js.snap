// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders UI as expected 1`] = `
<div
  className="not-found"
  role="main"
>
  <h1
    className="not-found-h1"
  >
    Page Not Found!
  </h1>
</div>
`;
